lib/config/virtual_container_host.go

```diff
@@ -214,6 +216,16 @@ func (t *VirtualContainerHostConfigSpec) SetMoref(moref *types.ManagedObjectRefe
 	}
 }
 
+// SetIsCreating sets the ID of the VCH to a constant, to identify the creating VCH VM before the VM moref can be set into this property
+func (t *VirtualContainerHostConfigSpec) SetIsCreating() {
```

Comment:
No, she sets `t.ExecutorConfig.ID` to this constant stand-in value and it eventually gets overwritten with an actual ID, I believe. So it's set like that until that field receives a real value, at which point the ContainerVM is created.

I think it might be better to use a dedicated bool rather than to overload the `ExecutorConfig.ID` field's use like this, though, to prevent the confusion that @sflxn is having.